cypress/e2e/home_page.cy.ts

```
@@ -0,0 +1,21 @@
+    it(`Should display correctly on ${name} viewport`, () => {
+      cy.viewport(width, height);
+      cy.visit('/');
+      cy.get('[data-testid="hero"]').should('be.visible');
+      cy.scrollTo('bottom', { duration: 4000 });
+      cy.waitForNetworkIdle(5000);
+      cy.matchImageSnapshot(`home-page-${name}`);
+    });
+  });
+});
+
+export {};
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)